README.md

```diff
@@ -85,30 +85,40 @@ Now in the server repository, push your server application to Cloud Foundry
 cf push
 ```
 
-Once the deployment is success, your backend server should be running on the cloud and able to classify the different kind of waste once the model finish training. Now let's go ahead and create our mobile app to use this classifier.
+Once the deployment is succeeds, your backend server should be running on the cloud and able to classify the different kinds of waste once the model finishes training. Please take a note with your server application's endpoint as you will need it in the next step. Now let's go ahead and create our mobile app to use this classifier.
```

Comment:
s/is succeeds/succeeds/
s/should be/will be/
s/on the cloud/in the cloud/
maybe s/able/be able/  (?)
s/take a note with/take note of/  (?)